tests/certification/bindings/azure/eventhubs/components/binding/iothub/eventhubs.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: azure-eventhubs-binding
+  namespace: default
+spec:
+  type: bindings.azure.eventhubs
+  version: v1
+  metadata:
+    - name: enableEntityManagement
+      value : true
+    - name: connectionString
+      secretKeyRef:
+        name: AzureIotHubEventHubConnectionString
+        key: AzureIotHubEventHubConnectionString 
+    - name: credentials
+      secretKeyRef:
+        name: AZURE_CREDENTIALS
+        key: AZURE_CREDENTIALS    
+    - name: storageAccountName
+      secretKeyRef:
+        name: AzureBlobStorageAccount
+        key: AzureBlobStorageAccount
+    - name: storageAccountKey
+      secretKeyRef:
+        name: AzureBlobStorageAccessKey
+        key: AzureBlobStorageAccessKey
+    - name: storageContainerName
+      secretKeyRef:
+        name: iothub-bindings-integration-test
```

Comment:
I'm not sure what this value needs to be -- maybe it does not need to be a secret and you can just pick a hardcoded value for the container?

This secret is not provided by the test infrastructure, so you cannot use it.